website/versioned_docs/version-v3.9.x/expansion.md

```diff
@@ -0,0 +1,218 @@
+---
+id: expansion title: Expansion
+---
+
+> 🚧 This feature is in _alpha_ stage and it is not enabled by default.
```

Comment:
can we add the flag to enable this?

Reply:
https://github.com/open-policy-agent/gatekeeper/pull/2062/commits/0b6c5626b1aed18031497bc2de80fdb6cb42a2b7